doc/source/_includes/rllib/rlmodules_rollout.rst

```diff
@@ -0,0 +1,5 @@
+..
+    .. admonition:: This doc is related to the RLModules API.
+
+        Disregard this doc if you do not explicitly want to use RLModules. The RLModules API is experimental and
```

Comment:
We could probably just say something to the effect of RLModules is an experimental API. I think this gets across the point overall. Its implied what experimental means.

Reply:
Can you provide a complete sentence to replace this with?

Reply:
The RLModules api is currently in the experimental phase of its rollout.

Reply:
Thanks!

Reply:
hmm while we're at it, is this a whole separate doc page right here? If so, why?

Reply:
It's an include. It's basically a banner that alerts users that they are looking at something experimental.
Angelina told me to make this basically the first thing people read - if it's not meant for John Doe, put it first.

CC @kouroshHakha to make you aware of this banner